mandatory/4-tax.js

```diff
@@ -17,7 +20,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
```

Comment:
good job. just as a suggestion you can use the first function inside the second function.

function addTaxAndFormatCurrency(price) {
  return "£"+(calculateSalesTax(price)).toFixed(2) 
}